draft-patton-cfrg-vdaf.md

```diff
@@ -868,8 +876,21 @@ def expand_into_vec(Prg, Field,
                     seed: Bytes,
                     info: Bytes,
                     length: Unsigned) -> Vec[Field]:
-    # TODO Specify a method for mapping a pseudorandom byte string
-    # to a sequence of field elemetns.
+    L = Field.ENCODED_SIZE
+    P = Field.EXPANDED_PAD_SIZE
+    len_in_bytes = length * L + P
+    uniform_bytes = Prg.expand(seed, info, len_in_bytes)
+
+    vec = []
+    r = OS2IP(uniform_bytes[:P])
+    for i in range(P, len(uniform_bytes), L):
+        vec.append(Field(r))
+        r /= Field.MODULUS
+        if L < P:
+            r <<= L * 8
+            r &= ((1 << P) - 1)
+        r |= O2SIP(uniform_bytes[i:i+L])
```

Comment:
As written above, this is not needed in the last iteration, so we can save L random bytes.